README.md

```diff
@@ -91,28 +91,28 @@ kermit [-h] [-v] [-d] [-c config] [-t title] [-w workdir] [-e command]
 [-e command] sets the command to execute in terminal
 ```
 
-## Key Bindings
-
-| Key                         | Action                         |
-| --------------------------- | ------------------------------ |
-| `ctrl-alt-[c]`              | copy to clipboard              |
-| `ctrl-alt-[v]`              | paste from clipboard           |
-| `ctrl-alt-[t]`              | open a new tab                 |
-| `ctrl-alt-[r]`              | reload configuration file      |
-| `ctrl-alt-[d]`              | load the default configuration |
-| `ctrl-alt-[q]`              | exit the terminal              |
-| `ctrl-alt-[k][up]`          | increase font size             |
-| `ctrl-alt-[j][down]`        | decrease font size             |
-| `ctrl-alt-[=]`              | reset font size to default     |
-| `ctrl-alt-[return]`         | open a new tab                 |
-| `ctrl-alt-[num]`            | switch to the tab number       |
-| `ctrl-alt-[l][right][pgup]` | switch to the next tab         |
-| `ctrl-alt-[h][left][pgdn]`  | switch to the previous tab     |
-| `ctrl-alt-[w][backspace]`   | close the selected tab         |
-
-• Key bindings (`ctrl-alt`) can be set to `ctrl-shift` with using the config file.
-
-• Default key bindings (`ctrl-alt`) might conflict with your desktop environments window shortcuts. To solve this issue, key bindings should be changed to `ctrl-shift`.
+## Default Key Bindings
+
+| Key                                    | Action                            |
+|----------------------------------------|-----------------------------------|
+| `ctrl` + `alt` + `c`                   | Copy to clipboard                 |
+| `ctrl` + `alt` + `v`                   | Paste from clipboard              |
+| `ctrl` + `alt` + `t`                   | Open a new tab                    |
+| `ctrl` + `alt` + `r`                   | Reload configuration file         |
+| `ctrl` + `alt` + `d`                   | Load default configuration        |
+| `ctrl` + `alt` + `q`                   | Exit the terminal                 |
+| `ctrl` + `alt` + `k`/`up`              | Increase font size                |
+| `ctrl` + `alt` + `j`/`down`            | Decrease font size                |
+| `ctrl` + `alt` + `equals`/`plus`       | Reset font size to default        |
+| `ctrl` + `alt` + `return`              | Open new tab                      |
```

Comment:
Not sure if this is correct, is seems like the new default is ctrl+alt+t

Reply:
The binding was set up incorrectly. Both should work now.